labs/iot/gps-lab/steps/set-up-azure-maps.md

```diff
@@ -0,0 +1,23 @@
+# Set Up Azure Map
+
+In this step you will set up Azure Map.
+
+## Steps
+
+1. Go to [Azure portal]('https://portal.azure.com') and login to your subscription. 
+2. Create a resource for Azure Maps  ![Azure Map](../images/azure-map-create-1.png)
+3. Select Azure Map![Azure Map](../images/azure-map-create-2.png)
+4. Note down the Primary Key and in [application.js](../server/code/static/js/application.js) and replace <Azure_Map_Subscription_Key> with the primary key
```

Comment:
How? This file is on GitHub so how is it updated? Should they clone the repo?

Reply:
I think when we say "replace this" is is evident that we are talking about the local copy.